androidApp/src/androidTest/java/com/projectnewm/interactions/BottomBarInteractions.kt

```diff
@@ -0,0 +1,24 @@
+package com.projectnewm.interactions
+
+import androidx.compose.ui.test.SemanticsNodeInteraction
+import androidx.compose.ui.test.junit4.ComposeTestRule
+import androidx.compose.ui.test.onNodeWithText
+
+fun onBottomBar(composeTestRule: ComposeTestRule, actions: BottomBarInteractions.() -> Unit) {
+    actions(BottomBarInteractions(composeTestRule))
+}
+
+class BottomBarInteractions(private val composeTestRule: ComposeTestRule) {
+    val homeButtonInteraction: SemanticsNodeInteraction
+        get() = composeTestRule.onNodeWithText("Home")
+
+    val walletButtonInteraction: SemanticsNodeInteraction
+        get() = composeTestRule.onNodeWithText("Wallet")
+
+    val starsButtonInteraction: SemanticsNodeInteraction
+        get() = composeTestRule.onNodeWithText("Stars")
+
+    val tribeButtonInteraction: SemanticsNodeInteraction
+        get() = composeTestRule.onNodeWithText("Tribe")
+}
```

Comment:
Need to make these finders less ambiguous

Reply:
and to use string resources

Reply:
Similar to how we do the theme, maybe we can define strings for now or we can also do
```
data class TestStrings(v : String) {
   data class HomeScreen("Home") : TestStrings
}
```

Reply:
I ended up pulling them from resources in a newer commit